src/utils/daos_autotest.c

```diff
@@ -27,8 +27,15 @@
 /** Input arguments passed to daos utility */
 struct cmd_args_s *autotest_ap;
 
+/* The start and end time in ns for individual test*/
+uint64_t	t_start, t_end;
+char		iops[32];
```

Comment:
Nit, seems a little be short if some unexpected iops value are given to the OUTPUT_IOPS macro. This could lead to some buffer overflow exploit.